modules/social_features/social_group/social_group.services.yml

```diff
@@ -49,3 +49,9 @@ services:
     arguments: []
     tags:
       - { name: cache.context }
+
+  social_group.current_group:
+    class: Drupal\social_group\CurrentGroupService
+    arguments:
+      - '@entity_type.manager'
+      - '@group.group_route_context'
```

Comment:
We can create an Interface for every things we create but in that case I dont see the point. If i s required I'm happy to comply otherwise i think it s unnecessary artefact.

Reply:
I agreed in my message that an interface may not be necessary here. We should still make sure that we use autowiring and define our services in a way so that this class can be used by another service that uses autowiring. That's what my suggested code snippet achieves.

Reply:
Fixed